vault/logical_system_quotas.go

```diff
@@ -193,15 +193,9 @@ func (b *SystemBackend) handleRateLimitQuotasUpdate() framework.OperationFunc {
 				return logical.ErrorResponse("invalid mount path %q", mountPath), nil
 			}
 
-			var newMountPath string
-			if me.Table == mountTableType {
-				newMountPath = me.Path
-			} else {
-				newMountPath = me.Table + "/" + me.Path
-			}
-
-			pathSuffix = strings.TrimSuffix(strings.TrimPrefix(mountPath, newMountPath), "/")
-			mountPath = newMountPath
+			mountAPIPath := me.APIPathNoNamespace()
```

Comment:
Would it make sense to add the pathSuffix line to this function as well? I am suggesting it because I am not sure if this function is used for other reasons than finally extracting pathSuffix.